index.bs

```diff
@@ -5410,6 +5415,83 @@ started</dfn> steps given |navigable| and |navigation status|:
 
 </div>
 
+#### The browsingContext.downloadEnd Event #### {#event-browsingContext-downloadEnd}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        browsingContext.DownloadEnd = (
+          method: "browsingContext.downloadEnd",
+          params: browsingContext.DownloadEndParams
+        )
+
+        browsingContext.DownloadEndParams = {
+          (
+            browsingContext.DownloadCanceledParams //
+            browsingContext.DownloadCompleteParams
+          )
+        }
+
+        browsingContext.DownloadCanceledParams = (
+          status: "canceled",
+          browsingContext.BaseNavigationInfo
+        )
+
+        browsingContext.DownloadCompleteParams = (
+          status: "complete",
+          filepath: text / null,
+          browsingContext.BaseNavigationInfo
+        )
+      </pre>
+   </dd>
+</dl>
+
+<div algorithm>
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi download finished</dfn>
```

Comment:
Maybe rename the trigger as well to match the new Event name?

Reply:
done